docs/API_POLICY.md

```diff
@@ -0,0 +1,39 @@
+# Hub API Policy
+This document proposes a policy regarding making API updates in this repo.
+
+There are 2 types of APIs:
+## 1. Versioned APIs: 
+- These are the APIs which are exposed for external users to use and integrate with their applications.
+- These will have support provided by the Hub team.
+- These are the APIs which return hub resource data.
+- These are versioned, the version of the APIs will be prefixed to the API route. 
+Eg. resource API - /v1/resources
```

Comment:
I think we should also document somewhere which API's are versioned and which are not, may be here as it would help the user as well